include/nng/supplemental/http/http.h

```diff
@@ -305,10 +305,6 @@ typedef struct nng_http_handler nng_http_handler;
 // Upon completion, the handler should store an nng_http_res * as the
 // first output using nng_aio_set_output.  If it does not do so, or supplies
 // NULL, then it must send a response itself.
-//
-// The callback should return 0 in most circumstances; if it returns anything
-// other than 0 then the connection is terminated (after possibly sending
-// a 500 error response to the client.)
```

Comment:
Why did you remove this comment.  I think it's accurate, no?

Reply:
I see the problem -- the issue is the word "return".
This should have have been reworded as:

"The callback should complete with a result of 0 most circumstances.  If it completes with an error,
then the connection is terminated, after possibly sending a 500 error response to the client"

Reply:
The critical information we need to convey is that a non-successful completion of the callback closes the connection (maybe returning a 500 error code as well.)

Reply:
Agreed! I updated the pull request with the suggested changes.